models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -322,11 +322,13 @@ FROM
     ('sand-the-sandbox', 'ethereum', 'SAND', 0x3845badade8e6dff049820680d1f14bd3903a5d0, 18),
     ('sdao-singularitydao', 'ethereum', 'SDAO', 0x993864e43caa7f7f12953ad6feb1d1ca635b875f, 18),
     ('sent-sentinel', 'ethereum', 'SENT', 0xa44e5137293e855b1b7bc7e2c6f8cd796ffcb037, 8),
+    ('seed-garden', 'ethereum', '$SEED', 0x5eed99d066a8caf10f3e4327c1b3d8b673485eed, 18),
```

Comment:
should symbol be just `SEED`?
https://etherscan.io/token/0x5eed99d066a8CaF10f3E4327c1b3D8b673485eED

Reply:
Looks like when I submitted it to coinpaprika, I included the '$' I've requested they updated that. I saw it failed a test, but is that just a benign check?

Reply:
the test assumes the coinpaprika symbol is correct. if you updated there, we should be good to update here after. it's also not a huge deal, it can be updated later in the spell

Reply:
Sounds good. I've reverted to the original commit. Let me know if there are any other adjustments you'd like to see. Once again, I appreciate the input jeff-dude!